src/test/java/com/rackspace/ceres/app/services/MetadataServiceTest.java

```diff
@@ -465,4 +467,70 @@ private Granularity granularity(int minutes, int ttlHours) {
         .setWidth(Duration.ofMinutes(minutes))
         .setTtl(Duration.ofHours(ttlHours));
   }
+
```

Comment:
I would like to see the aforementioned tests:

1. omit
2. null
3. "" 

for all 3 tenant, metric, metricgroup => 9 tests